File: server/Models/VaultKeep.cs
namespace keepr.Models;

public class VaultKeep
{
  public int Id { get; set; }
  public DateTime CreatedAt { get; set; }
  public DateTime UpdatedAt { get; set; }
  public int KeepId { get; set; }
  public int VaultId { get; set; }
  public string CreatorId { get; set; }

}

File: client/src/models/VaultKeep.js
export class VaultKeep {
  constructor(data) {
    this.id = data.id
    this.createdAt = new Date(data.createdAt)
    this.updatedAt = new Date(data.updatedAt)
    this.keepId = data.keepId
    this.vaultId = data.vaultId
    this.vaultKeepId = data.vaultKeepId
    this.creatorId = data.creatorId

  }
}